R/fledgling.R

```diff
@@ -265,30 +265,43 @@ format_news_subsections <- function(news_list, header_level) {
     ),
   )
 
-  paste(lines, collapse = "\n\n")
+  paste_n_lines_only(lines, n = 1)
 }
 
 paste_news_lines <- function(lines, header_level) {
   lines <- unlist(lines, recursive = FALSE)
-  if (is_any_named(lines)) {
-    header_sign <- paste(rep("#", header_level), collapse = "")
-    sub_header <- function(x, header_sign) {
-      if (!nzchar(x)) {
-        ""
-      } else {
-        paste(header_sign, x, "\n\n")
-      }
-    }
-    lines <- purrr::imap_chr(
-      lines,
-      ~ sprintf(
-        "%s%s",
-        sub_header(.y, header_sign),
-        paste_news_lines(.x, header_level = header_level + 1)
-      )
+
+  subsections_present <- (is_any_named(lines))
+
+  if (!subsections_present) {
+    lines <- gsub("^- ", "\n- ", lines)
```

Comment:
this is a bit of a hack... sometimes there are new lines in the messages, but we've bulleted them previously so we can recognize items.